gcc/config/aarch64/aarch64-coff.h

```diff
@@ -34,9 +34,14 @@
 #define PTRDIFF_TYPE	"long long int"
 
 #define TARGET_64BIT 1
+#define TARGET_LONG_DOUBLE_64 1
```

Comment:
Why is TARGET_LONG_DOUBLE_64 used instead of TARGET_LONG_DOUBLE_128?

Reply:
64bit double is implemented in hardware, 128bit double must be emulated. MSVC uses 64bit double.